TFT/src/User/Menu/SendGcode.c

```diff
@@ -1,20 +1,44 @@
 #include "SendGcode.h"
 #include "includes.h"
 
-#define TERMINAL_MAX_CHAR NOBEYOND(600, RAM_SIZE*50, 4800)
-#define MAX_BUFF          20
-#define SCROLL_LINE       22
-#define SCROLL_PAGE       1
+#define TERMINAL_MAX_CHAR (NOBEYOND(600, RAM_SIZE*45, 4800))
```

Comment:
Is there any specific reason for reducing it?

Reply:
Due to this: 
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/blob/2a8527b826d98a11142f2671348f951734a20a0b/TFT/src/User/Menu/SendGcode.c#L662

Reply:
Sorry but I do not get it. Would you mind educate me a little?

Reply:
There is no more `Rcv: ` prepended to received text and the sent text is Prepended with `>>` instead of `Send: `. This increased the buffer capacity for each received line by 5 characters and for each sent line by 4 characters for displaying the actual data.

Reply:
Ah! Thanks for the insight.. 
However I wouldn't remove completely "Rcv:". As a suggestion to this new syntax, maybe "<" for receive and ">" for send could be used. I think it would be easier for the eye to catch where send and receive starts in case of messages that are displayed on more than one line. It's only a suggestion.

Reply:
There is no need to prepend `<` on the received text on these small screens as it is easier to read more text in a single line than breaking it down into multiple lines. `>>` is used because the sent commands are mostly smaller and barely fill half-line.